crates/typst-library/src/visualize/gradient.rs

```diff
@@ -706,6 +706,46 @@ impl Gradient {
         }
     }
 
+    /// Returns the radius of this gradient.
+    #[func]
+    pub fn center(&self) -> Option<Axes<Ratio>> {
+        match self {
+            Self::Linear(_) => None,
+            Self::Radial(radial) => Some(radial.center),
+            Self::Conic(conic) => Some(conic.center),
+        }
+    }
+
+    /// Returns the radius of this gradient.
```

Comment:
For radius, focal-center and focal-radius, it'd be nice to specify that this returns `none` if the gradient isn't radial.